style.css

```diff
@@ -0,0 +1,31 @@
+div {
+  font-style: italic;
+  text-align: center;
+}
+@media (max-width:320px) {
+  div { font-size: .5em; color: blue } /* smartphones4 iPhone, portrait 480x320 phones */
+   }
+   
+@media (min-width:320px) {
+  div { font-size: 1em; color: green; } /* smartphones4 iPhone, portrait 480x320 phones */
+   }
+
+@media (min-width:481px) {
+  div { font-size: 2em; color: red; } /* portrait e-readers (Nook/Kindle), smaller tablets @ 600 or @ 640 wide. */
+  }
+
+@media (min-width:641px) {
+  div { font-size: 3em; color: black; } /* portrait tablets, portrait iPad, landscape e-readers, landscape 800x480 or 854x480 phones */
+  }
+  
+@media (min-width:961px) {
+  div { font-size: 4em; color: yellow; } /* tablet, landscape iPad, lo-res laptops ands desktops */
+  }
+  
+@media (min-width:1025px) {
+  div { font-size: 5em; color: orange; } /* big landscape tablets, laptops, and desktops */
+  }
+  
+@media (min-width:1281px) {
+  div { font-size: 6em; color: grey;} /* hi-res laptops and desktops */
```

Comment:
Codacy found an issue: [Expected single space before "}" of a single-line block (block-closing-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/css17__/pullRequest?prid=5991486)